src/main/java/de/dennisguse/opentracks/chart/ChartView.java

```diff
@@ -379,6 +397,10 @@ public void setReportSpeed(boolean value) {
     }
 
     public boolean applyReportSpeed() {
+        if (!showPaceOrSpeed) {
+            return true;
```

Comment:
Should setEnabled(false) on speedSeries, right/

Reply:
@dennisguse technically this was already happening as part of `setShowPaceOrSpeed`, so it was not necessary. But I think it's probably best to consolidate that here, so I'll move it over.